src/shared/scrapers/CA/index.js

```diff
@@ -1,15 +1,33 @@
+import assert from 'assert';
+import datetime from '../../lib/datetime/old/index.js';
 import * as fetch from '../../lib/fetch/index.js';
-import * as parse from '../../lib/parse.js';
-import * as transform from '../../lib/transform.js';
 import * as geography from '../../lib/geography/index.js';
-import datetime from '../../lib/datetime/old/index.js';
+import * as parse from '../../lib/parse.js';
 import * as rules from '../../lib/rules.js';
 
 // Set county to this if you only have state data, but this isn't the entire state
 // const UNASSIGNED = '(unassigned)';
 
+const countryLevelMap = {
+  Canada: 'iso1:CA',
+  'British Columbia': 'iso2:CA-BC',
+  'New Brunswick': 'iso2:CA-NB',
+  'Newfoundland and Labrador': 'iso2:CA-NL',
+  'Northwest Territories': 'iso2:CA-NT',
+  'Nova Scotia': 'iso2:CA-NS',
+  'Prince Edward Island': 'iso2:CA-PE',
+  'Repatriated travellers': '-',
```

Comment:
We'll handle these correctly (not even sure what correctly would do here, still add to the canada total?)?